loom/graphics/gfxQuadRenderer.cpp

```diff
@@ -95,124 +95,127 @@ void QuadRenderer::submit()
 
     if (tinfo.handle != -1)

```

Comment:
AFAIK it still needs to do `currentIndexBufferIdx += quadCount * 6;` at the end, so I thought this was the easiest / best compromise instead of duplicating code or rearranging the code more.